.github/workflows/docs-ci.yaml

```diff
@@ -48,8 +48,7 @@ env:
   SPHINXOPTS: -n -W --keep-going
 
 jobs:
-  build-conda:
-    if: inputs.environment-file != ''
+  build:
```

Comment:
Might be a nice safe-guard to have an early step that checks for both `inputs.environment-file` and `inputs.pip-install-target` being `== ''` and bails out with an error about the config being wrong. 

(I'm thinking of situations like somebody typo-ing in the config or something…)

Reply:
Same for having both `environment-file` and `pip-install-target` for whatever reason. It'll probably work fine but we shouldn't have a use case for that.

I'll make a separate PR after merging this one.

Reply:
https://github.com/nextstrain/.github/pull/111